testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/LoginTest.java

```diff
@@ -1029,4 +1033,35 @@ public void testExecuteActionIfSessionExists() {
         }
 
     }
+
+    @Test
+    public void testFrontChannelLogoutCustomCSP() throws Exception {
```

Comment:
No, it's not possible. For this test I need the chrome/firefox driver to correctly follow the CSP directives. If the test uses default browser it is always OK because the CSP directives are not taken into account. So I need the forms or the wenauthn CI run (the ones that uses `-Dbrowser=chrome`). I remember I initially added the test in another class to realize that it was not tested in CI. If you want I can add it in a complete new class nut in forms folder (something like `CSPLoginTest` for example).